transport/endpoint.go

```diff
@@ -1,12 +1,17 @@
 package transport
 
-import "github.com/aws/smithy-go"
+import (
+	"net/http"
+	"net/url"
+	"github.com/aws/smithy-go"
+)
 
-// Endpoint is a Smithy endpoint.
+// Endpoint is the endpoint object returned
+// by Endpoint resolution V2
 type Endpoint struct {
```

Comment:
**question**: We have the `endpoints` package should this not go in there? Or does that create issues with the package layout w.r.t `rulesfn`?